ci2/Makefile

```diff
@@ -30,7 +30,7 @@ build:
 endif
 
 run-local:
-	python3 ci/ci.py
+	PYTHONPATH=../batch python3 ci/ci.py
```

Comment:
this should probably be `PYTHONPATH=../batch:$PYTHONPATH`, no? How does python find the other installed libraries?